package.json

```diff
@@ -149,11 +155,11 @@
     "ts-jest": "^26.4.4",
     "ts-loader": "^8.0.11",
     "ts-node": "^9.0.0",
-    "typescript": "^4.0.5"
+    "typescript": "^4.1.1-rc"
```

Comment:
Is this required for the react 17 upgrade?

Reply:
No; I installed this trying to figure out why everything broke 😃 Then I decided to leave it 🤷‍♂️